plugins/basic/nexus-timeline-plugin/src/main/java/org/sonatype/nexus/timeline/feeds/internal/DefaultFeedContentRenderer.java

```diff
@@ -54,7 +54,8 @@ public String getTitle(final FeedEvent evt) {
     if (title != null) {
       return title;
     }
-    return plainTextRenderer.getTitle(evt);
+    // TODO: Some human-readable fallback?
+    return evt.getEventType() + ":" + evt.getEventSubType();
```

Comment:
No, it contained an omelette of all feed titles, I wanted to move that monolithic method out of rendered. Also, what "renderer" has to do how feed title is composed?